### PR TITLE
Martin

### PR DESCRIPTION
**Pull request by martinfrnka**
I'd like to incorporate this part of story, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: czech/dvere.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Ochutnal jsi maršmelouny. Protože ti zachutnaly, baštíš a baštíš až se ti z toho
```
Comment (casey-collab) [inline czech/dvere.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to address the comments above, feel free to re-open it.

**Pull request closed without merging**